libs/openFrameworks/graphics/ofImage.cpp

```diff
@@ -209,23 +214,35 @@ static bool loadImage(ofPixels_<PixelType> & pix, const of::filesystem::path& _f
 		return ofLoadImage(pix, ofLoadURL(_fileName.string()).data);
 	}
 
-	auto fileName = ofToDataPath(_fileName, true);
+	auto fileName = ofToDataPathFS(_fileName, true);
```

Comment:
variables names like "filename" should better represent the underlying object, in this case "filepath"

Reply:
Yeah I just left as it was in OF. later on there is a function which calls filename, so it is the same for me
FreeImage_GetFIFFromFilename(fileName)